src/entries/popup/components/CommandK/CommandKToolbar.tsx

```diff
@@ -105,7 +105,7 @@ export function CommandKToolbar({
                   whileTap={{ scale: 0.925 }}
                 >
                   <Inline alignVertical="center" space="10px" wrap={false}>
-                    <Inline alignVertical="center" space="4px">
+                    <Inline wrap={false} alignVertical="center" space="4px">
```

Comment:
This change solved this bug i had

<img width="372" alt="Screenshot 2024-05-06 at 15 17 35" src="https://github.com/rainbow-me/browser-extension/assets/53529533/2cc51b1c-9b33-4157-815e-918f2382bd9e">